var/spack/repos/builtin/packages/stream/package.py

```diff
@@ -17,6 +17,7 @@ class Stream(MakefilePackage):
     version('5.10')
 
     variant('openmp', default=False, description='Build with OpenMP support')
+    variant('STREAM_ARRAY_SIZE', default=2500000000, description='Please note that this is system specific setting, please modify it if you want to compile it with different STREAM ARRAY SIZE other than "2500000000". Currently this variant has effect only with AOCC compiler.General recommendation is that STREAM_ARRAY_SIZE must be at least 4x the size of the sum of all the last-level caches in the system')
```

Comment:
```suggestion
    variant('STREAM_ARRAY_SIZE', default=2500000000, description='Please note that this is system specific setting, please modify it if you want to compile it with different STREAM ARRAY SIZE other than "2500000000". Currently this variant has effect only with AOCC compiler. General recommendation is that STREAM_ARRAY_SIZE must be at least 4x the size of the sum of all the last-level caches in the system')
```
You may want to break this up on multiple lines as well.

Reply:
I would also make this lowercase to be consistent with other variant names.